File: core/src/main/java/com/a65apps/core/interactors/reminders/BirthdayReminderRepository.java
package com.a65apps.core.interactors.reminders;

import java.util.GregorianCalendar;

import io.reactivex.rxjava3.annotations.NonNull;

public interface BirthdayReminderRepository {
    boolean enableBirthdayReminder(@NonNull String personId, @NonNull String fullName, @NonNull String date);
    boolean disableBirthdayReminder(@NonNull String personId);
    boolean isBirthdayReminderEnabled(@NonNull String personId);
}

File: core/src/main/java/com/a65apps/core/interactors/reminders/BirthdayReminderRepository.kt
package com.a65apps.core.interactors.reminders

interface BirthdayReminderRepository {
    fun enableBirthdayReminder(personId: String, fullName: String, date: String): Boolean
    fun disableBirthdayReminder(personId: String): Boolean
    fun isBirthdayReminderEnabled(personId: String): Boolean
}
